lib/metrics.coffee

```diff
@@ -40,7 +44,10 @@ start_pause_detector = (source, sleep_time_ms, pause_threshold_ms) ->
   setInterval pause_fn, sleep_time_ms
 
 log_pauses = (source) ->
-  console.error kayvee.formatLog(source, kayvee.INFO, 'PauseMetric', { type: "gauge", value : module.exports._last_period_pause_ms})
+  console.error kayvee.formatLog source, kayvee.INFO, 'PauseMetric',
```

Comment:
I started to do that, but it seemed too complicated for how short this script is.